src/configuration-storage/chrome-storage.ts

```diff
@@ -0,0 +1,36 @@
+import { IAsyncWriteSyncReadConfigurationStore } from '@eppo/js-client-sdk-common';
+
+/*
+ * Removing stale keys is faciliating by storing the entire configuration into a single
+ * key in local storage. This is done by serializing the entire configuration object
+ * into a string and then storing it in local storage. When retrieving the configuration,
+ * we first check if the configuration exists in local storage. If it does, we deserialize
+ * the configuration from the string and return it. If the configuration does not exist,
+ * we return null.
+ */
+export class ChromeStorageBackedConfigurationStorage
+  implements IAsyncWriteSyncReadConfigurationStore
+{
+  private chromeStorageKey = 'eppo-configuration';
+  private memoryCache: Record<string, any> = {};
+
+  public isInitialized(): boolean {
+    return this.memoryCache.length > 0;
+  }
+
+  // Serve results from the memory cache to avoid slow and sync local storage operations
+  public get<T>(flagKey: string): T {
+    return this.memoryCache[flagKey];
+  }
```

Comment:
```suggestion
  public get<T>(key: string): T {
    return this.memoryCache[key];
  }
```

Nit: this class does not need to know the keys are flag keys